src/components/ec-panel/ec-panel.vue

```diff
@@ -0,0 +1,96 @@
+<template>
+  <aside
+    v-if="showPanel"
+    class="ec-panel"
+  >
+    <header class="ec-panel__header">
+      <div class="ec-panel__header-icons">
+        <ec-icon
+          class="ec-panel__close"
+          name="simple-close"
+          :size="24"
+          @click="$emit('close')"
+        />
+      </div>
+
+      <div>
+        <slot name="header" />
+      </div>
+    </header>
+
+    <main>
+      <slot name="main" />
+    </main>
+  </aside>
+</template>
+
+<script>
+import EcIcon from '../ec-icon';
+
+
+export default {
+  name: 'EcPanel',
+  components: {
+    EcIcon,
+  },
+  model: {
+    prop: 'showPanel',
+    event: 'close',
+  },
+  props: {
+    showPanel: {
+      type: Boolean,
+      default: false,
+    },
+  },
+
+};
+</script>
+
+<style lang="scss">
+@import '../../scss/settings/colors/index';
+@import '../../scss/tools/index';
+
+$ec-panel-close-btn-fill: $level-4-interactive-elements !default;
+$ec-panel-close-btn-fill-hover: $level-4-tech-blue !default;
+
+.ec-panel {
+  display: block;
+  width: 100%;
+  max-width: 352px;
```

Comment:
I'm curious where we get this magic number from. maybe extract it into variable and give it name others can better understand.

Reply:
Got it from UX team. There is nowhere on documentation, just the max-width of the side panel. Shall I make it a variable even though we might not reuse it somewhere else?